src/transaction/log_page_buffer.c

```diff
@@ -115,77 +116,82 @@
 #if !defined(SERVER_MODE)
 #define pthread_mutex_init(a, b)
 #define pthread_mutex_destroy(a)
-#define pthread_mutex_lock(a)	0
+#define pthread_mutex_lock(a) 0
 #define pthread_mutex_unlock(a)
 static int rv;
-#undef  COND_INIT
+#undef COND_INIT
 #define COND_INIT(a)
-#undef  COND_BROADCAST
+#undef COND_BROADCAST
 #define COND_BROADCAST(a)
-#undef  COND_DESTROY
+#undef COND_DESTROY
 #define COND_DESTROY(a)
 #endif /* !SERVER_MODE */
 
-#define logpb_log(...) if (logpb_Logging) _er_log_debug (ARG_FILE_LINE, "LOGPB: " __VA_ARGS__)
-#define log_archive_er_log(...) \
-  if (prm_get_bool_value (PRM_ID_DEBUG_LOG_ARCHIVES)) _er_log_debug (ARG_FILE_LINE, __VA_ARGS__)
+#define logpb_log(...)     \
```

Comment:
There are too many indentation changes and they don't seem to be done by indent too. Is this done automatically by your IDE?